relayer/strategies.go

```diff
@@ -257,6 +258,19 @@ func relayUnrelayedPackets(ctx context.Context, log *zap.Logger, src, dst *Chain
 			return false
 		}
 
+		// If we encounter an error that suggest node configuration issues, log a more insightful error message.
+		if strings.Contains(err.Error(), "Internal error: transaction indexing is disabled") {
+			log.Warn(
+				"Remote server needs reconfigured.",
```

Comment:
No punctuation at the end of log messages. Maybe reword to "Remove server needs to enable transaction indexing"?